plasma_framework/migrations/140_payment_exit_game.js

```diff
@@ -28,8 +28,8 @@ module.exports = async (
         PAYMENT_TX_TYPE,
         config.frameworks.safeGasStipend.v1,
     ];
-    const paymentExitGame = await deployer.deploy(PaymentExitGame, paymentExitGameArgs);
-
+    const paymentExitGame = await deployer.deploy(PaymentExitGame);
+    paymentExitGame.init(paymentExitGameArgs);
```

Comment:
so the questin is should this be allowed only from maintainer? it wasn't the case via constructor.